website/blog/2021-02-22-7.13.0.md

```diff
@@ -0,0 +1,255 @@
+## Highlights
+
+### Top-level `targets` option ([#12189](https://github.com/babel/babel/pull/12189), [RFC](https://github.com/babel/rfcs/pull/2))
+
+The `targets` option is the true power behind `@babel/preset-env`: it allows users the ability to specify their target environments which allows the preset to automatically choose which syntax to transform and what polyfills to inject.
+
+Since releasing `@babel/preset-env`, we've learned that plugins _themselves_ can also benefit from knowing your targets. Currently, this can a bit cumbersome, since you'd have to specify your targets twice (for example, if you are using our [new polyfill plugins](https://github.com/babel/babel-polyfills). By introducing `targets` as a top level option, you now only have to specify your targets once:
+
+<table>
+<tr>
+<th>Old configuration</th>
+<th>New configuration</th>
+</tr>
+<tr>
+<td>
+
+```json
+{
+  "presets": [
+    ["@babel/preset-env", {
+      "targets": ">1%, not ie 11"
+    }]
+  ],
+  "plugins": [
+    ["polyfill-es-shims", {
+      "targets": ">1%, not ie 11"
+    }]
+  ]
+}
+```
+
+</td>
+<td>
+
+```json
+{
+  "targets": ">1%, not ie 11",
+  "presets": ["@babel/preset-env"],
+  "plugins": ["polyfill-es-shims"]
+}
+
+
+
+
+
+
+
+```
+
+</td>
+</tr>
+</table>
+
+We recommended converting your Babel config to use the new top-level `targets` option since:
+
+1. It has better handling of the [`esmodules: true`](https://babel.dev/docs/en/options#targetsesmodules) target. (it's intersected with the other targets, rather than replacing them)
+2. As we introduce additional `targets`-related support in our plugins, you'll automatically benefit from more optimized output!
+
+You can read the details about this new option in its [RFC](https://github.com/babel/rfcs/blob/master/rfcs/0002-top-level-targets.md).
+
+> 🔮 In the future we might explore moving `@babel/preset-env` into `@babel/core` so that you don't have to install an additional package to get started with Babel. This new option can be seen as the first step in that direction!
+
+### Top-level `assumptions` option ([#12219](https://github.com/babel/babel/pull/12219), [RFC](https://github.com/babel/rfcs/pull/5))
+
+Many of our plugins have a `loose` option, which enables Babel to generate smaller/faster output by making certain assumptions about your code and ignoring certain edge cases.
+
+However, `loose` has some problems that lead to user frustration/confusion: the term itself does nothing to describe how it affects the compiler's behavior, and worse, sometimes it must be set the same across different plugins to ensure everything compiles.
+
+To help fix these problems, we've added a new top level option: `assumptions`! Similar to the new `targets` option, every plugin will now receive the assumptions you've enabled, eliminating the need to set the option individually. This is really valuable since a plugin can be influenced by multiple assumptions, and an assumption can influence multiple plugins.
```

Comment:
I think the problem is described well, but I don't think it's clear what an "assumption" is at this point. The examples below help clear this up a bit, but I think it might be useful to provide a little bit more of an explanation in the intro. i.e. Why are they called assumptions? What are we making assumptions about?

Reply:
```suggestion
To help fix these problems, we've added a new top level option to tell Babel which assumptions it can make about your code: `assumptions`! Similar to the new `targets` option, every plugin will now receive the assumptions you've enabled, eliminating the need to set the option individually. This is really valuable since a plugin can be influenced by multiple assumptions, and an assumption can influence multiple plugins.
```

Is this better? I don't really know how to explain what an assumption is :sweat_smile:

Reply:
Yeah, that's a tough concept...I do think this is an improvement!